File: app/src/main/java/droiddevelopers254/droidconke/utils/DroidCoin.java
package droiddevelopers254.droidconke.utils;

import android.app.Application;
import android.content.Context;

import com.google.firebase.database.FirebaseDatabase;

public class DroidCoin extends Application {
    public  static Context context;
    @Override
    public void onCreate() {
        super.onCreate();
        FirebaseDatabase.getInstance().setPersistenceEnabled(true);
       context=getApplicationContext();
    }

}

File: app/src/main/java/droiddevelopers254/droidconke/utils/DroidCoin.kt
package droiddevelopers254.droidconke.utils

import android.app.Application
import android.content.Context

import com.google.firebase.database.FirebaseDatabase

class DroidCoin : Application() {
    override fun onCreate() {
        super.onCreate()
        FirebaseDatabase.getInstance().setPersistenceEnabled(true)
        context = applicationContext
    }

    companion object {
       lateinit var context: Context
    }

}
